docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Troubleshooting.md

```diff
@@ -58,6 +58,18 @@ Active = 'Y'
 Pending_approval = 'N'
 Password_expiry column value is later than today's date
 ```
+If your admin user has been deactivated due to inactivity, you can get past that using the following steps:
+
+1. Log into the to mysql backend with your admin credential
+2.  Make sure the `Active` column in the `users` MySQL table is set to `Y` for this user, if not you can do so with the following command:
+       ```
+       UPDATE users SET active = "Y" WHERE UserID = "admin";
+       ```
+3. Run the following command:
+   ```
+   DELETE FROM user_login_history WHERE UserID = "admin";
+   ```
+   ***Note:** Using this command will wipe out your account's login history*
```

Comment:
I might have found a better way a little while ago. I think instead of erasing history, we can manually add an entry like
```
INSERT INTO user_login_history (userID,Success,Page_requested) VALUE ('admin','Y','/');
```

This need to be tested but I find it a better alternative to deletion

Reply:
Hi @satvik-tha-god. Please let's try @ridz1208 suggestions and if goes well I will do prefer this solution instead of erasing the history of the account, I agree with both @christinerogers and @ridz1208.

Reply:
I made the changes

Reply:
Sorry for not being descriptive earlier, I did run the command on my database and everything seems to be working fine

I was able to login normally with no issues after running those commands